beatrix/src/test/resources/catalogs/testCatalogFixedTerm/versionedCatalogs/WaterUtility-v1.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ Copyright 2014 The Billing Project, Inc.
+  ~
+  ~ Ning licenses this file to you under the Apache License, version 2.0
+  ~ (the "License"); you may not use this file except in compliance with the
+  ~ License.  You may obtain a copy of the License at:
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+  ~ License for the specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<catalog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="CatalogSchema.xsd ">
+
+    <effectiveDate>2021-01-01T00:00:00+00:00</effectiveDate>
+    <catalogName>MyCatalog</catalogName>
+
+    <recurringBillingMode>IN_ADVANCE</recurringBillingMode>
+
+    <currencies>
+        <currency>USD</currency>
+    </currencies>
+
+    <units>
+        <unit name="liter"/>
+    </units>
+
+    <products>
+        <product name="PlumberInsurance">
+            <category>BASE</category>
+        </product>
+        <product name="Water">
+            <category>BASE</category>
+        </product>
+    </products>
+
+    <rules>
+        <changePolicy>
+            <changePolicyCase>
+                <policy>IMMEDIATE</policy>
+            </changePolicyCase>
+        </changePolicy>
+        <cancelPolicy>
+            <cancelPolicyCase>
+                <policy>IMMEDIATE</policy>
+            </cancelPolicyCase>
+        </cancelPolicy>
+    </rules>
+
+    <plans>
+     <plan name="plumber-insurance-monthly">
+            <product>PlumberInsurance</product>
+            <initialPhases>
+            </initialPhases>
+            <finalPhase type="FIXEDTERM">
```

Comment:
The xml are all misaligned - perhaps the one you copied from where already like that? In any case, if this is easy, can you reformat them - there are probably online formatter you could use or perhaps a plugin in eclipse?

Reply:
Fixed, sorry about that.